### PR TITLE
Update the README

### PR DESCRIPTION
**Pull request by sradco**
This PR updates the README with latest updates.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,16 +20,13 @@ information and custom configurations about the lvctl command, head to the
...
-  wget https://github.com/vladikr/logsviewer/releases/download/${LATEST_TAG}/lvctl
-  chmod +x lvctl
-  
-  # Or to build the lvctl binary from source
-  make build -C tools/lvctl/
-  cp tools/lvctl/bin/lvctl . 
+	$ git clone https://github.com/vladikr/logsviewer
+	$ cd logsviewer/
+	$ make build -C tools/lvctl/
+	$ mv tools/lvctl/bin/lvctl /usr/local/bin/
```
Comment (vladikr) [inline README.md:26]: :+1:

File: README.md [hunk 2]
```diff
@@ -1,13 +1,12 @@
-# Troubleshooting as a serivce
+# Troubleshooting as a service using Must-Gather
```
Comment (vladikr) [inline README.md:1]: Do you see a benefit in only committing to Must-Gathers?
I mean, right now we don't have any other reliable way, but in the general case, we could make it work with cluster dump or even connecting to a live cluster. I think plain Kubernetes doesn't operate on Must-Gathers.

Comment (sradco) [reply README.md]: @vladikr I updated based on your comment. Please review.

**Review by vladikr — APPROVED** [review]
Thank you!

**Pull request merged**